tools/c7n_oci/c7n_oci/filters/list.py

```diff
@@ -0,0 +1,41 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n.utils import type_schema
+from c7n.filters.core import Filter
+from c7n_oci.provider import resources
+
+class QueryFilter(Filter):
```

Comment:
there's a top level query key in the policy schema that would be better suited for what you're trying to do here

Reply:
yeah server side queries should be handled by the source. as is this filter appears non functional and can be removed.